<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Значение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Список ключей</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Ключ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="updateButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/SilkCompanion1/icons/reload.png</normaloff>:/images/SilkCompanion1/icons/reload.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>-------------</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="12" colspan="4">
    <widget class="QListWidget" name="keysListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="valueEdit">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="editButton">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/SilkCompanion1/icons/bullet_edit.png</normaloff>:/images/SilkCompanion1/icons/bullet_edit.png</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../core/frameWork/resources/framework_resources.qrc"/>
 </resources>
 <connections/>
</ui>
